src/main/java/org/jarvis4ops/helper/SlackHelper.java

```diff
@@ -140,5 +145,62 @@ public String composeWipBreachedMsg(HashMap<String, String> jiraMaxWipCountMap)
 		}
 		return jiraMaxWipBreachedDetailMsgSb.toString();
 	}
+	
+	public void getInvlaidTimes(HashMap<String, TimeBean> map) {
+		
+		Map<String, String> jiraMap = new HashMap<String, String>(10);
+		jiraMap.put("agupta", "Akash");
+		jiraMap.put("htomar", "Himanshu");
+		jiraMap.put("pkumar1", "Prabhat");
+		jiraMap.put("dkhandelwal", "Divyansh");
+		jiraMap.put("rarora", "Raghav");
+		jiraMap.put("bkaur", "Bikran");
+		jiraMap.put("jsahni@sapient.com", "Japneet");
+		jiraMap.put("pgoyal", "Pulkit");
+		jiraMap.put("aksharma", "Avinash");
+		jiraMap.put("hsrivastava", "Harshit");
+		jiraMap.put("hsinha", "Harshita");
+		jiraMap.put("vmathur", "Varuneshwar");
+		jiraMap.put("ayadav", "Antariksh");
+		jiraMap.put("marora", "Madhur");
+		jiraMap.put("sswami", "Sundeep");
+		jiraMap.put("ssingh1", "Sandeep");
+		jiraMap.put("rpandey", "Rahul");
+		jiraMap.put("sgupta1", "Sumit");
+		jiraMap.put("uagarwal", "Umang");
+		jiraMap.put("rjha", "Raman");
+		jiraMap.put("pkohli", "Prakash");
+		jiraMap.put("jtripathy", "Jitendra");
+		jiraMap.put("sgoel1", "Sonali");
+		jiraMap.put("vkumar1", "VikasKumar");
+		jiraMap.put("dgupta", "Deepak");
+		jiraMap.put("ibanerjee", "Ishika");
```

Comment:
Refactor this please - we dont need duplicate maps - populating the heap with bulk data.